### PR TITLE
Skier summary page enhancement

### PR DESCRIPTION
**Pull request by Jar0T**
Displayed data comes from database and added ability to choose specific lift

### CODE REVIEW

File: src/E-SkiLift/Models/SkierSummaryModel.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        public SkierSummaryModel(int ticketID, int liftID)
+        {
+            LiftUsageHistory[] liftUsageHistory = uow.LiftUsageHistory.GetLiftUsageHistoryByLiftIDAndTicketID(liftID, ticketID).ToArray();
+            LiftUsageHistory[] liftUsageHistoryLiftIDs = uow.LiftUsageHistory.GetLiftUsageHistoryByTicketID(ticketID).ToArray();
+
+            TicketID = ticketID;
+
+            SummaryEntryList = new List<SummaryEntry>();
+            for (int i = 0; i < liftUsageHistory.Length; i++)
```
Comment (syphzur) [inline src/E-SkiLift/Models/SkierSummaryModel.cs]: i think it's better to use foreach-in loop to keep code readable

**Review by syphzur — CHANGES_REQUESTED** [review]
please change for loops to foreach-in

**Comment by Jar0T** [comment]
Changed for loops to foreach and fixed bug that occured if user selected specific lift and returned from summary page and then wanted to see summary for another ticket that wasn't used at previously selected lift.

**Comment by syphzur** [comment]
> Changed for loops to foreach and fixed bug that occured if user selected specific lift and returned from summary page and then wanted to see summary for another ticket that wasn't used at previously selected lift.

mmmm mmmm mmm really nice

**Pull request merged**